test/api-digester/Outputs/stability-stdlib-abi.asserts.additional.swift.expected

```diff
@@ -1,4 +1,5 @@
 Class __SwiftNativeNSData has been removed
+Func _boxUTF8CString(_:_:) is a new API without @available attribute
```

Comment:
I tried putting @ available on it, but it still complained. Not sure if it just doesn't work with this sort of silgen name hackery.

Reply:
`@available` ought to work. What did you try exactly?

Reply:
I've already forgotten, I'll make a note to give it another shot